src/app/components/Footer/index.stories.jsx

```diff
@@ -2,4 +2,12 @@ import React from 'react';
 import { storiesOf } from '@storybook/react'; // eslint-disable-line import/no-extraneous-dependencies
 import Footer from './index';
 
-storiesOf('Footer', module).add('default', () => <Footer />);
+const link = {
+  href: 'Welcome to the BBC',
```

Comment:
I think it would be useful to have at least 4 links here. This is so that the grid layout can be viewed in Storybook.